### PR TITLE
Exposed shared storage quota for workspace

### PR DESCRIPTION
**Pull request by oliver7598**
# PR for issue
#1267 

## What is being addressed
Currently when shared storage is created as part of the base workspace it is set to have a quota for 50.

## How is this addressed
- Base workspace version number iterated 
- Optional property added to `template_schema.json` called `shared_storage_quota` with a default value of 50.
- Property mapped to workspace storage account file share vm-shared-storage.

### CODE REVIEW

**Review by jjgriff93 — APPROVED** [review]
Looks good!

File: templates/workspaces/base/terraform/variables.tf [hunk 1]
```diff
@@ -8,6 +8,11 @@ variable "tre_resource_id" {
   description = "Resource ID"
 }
 
+variable "shared_storage_quota" {
```
Comment (martinpeck) [inline templates/workspaces/base/terraform/variables.tf:11]: can you add a description to this variable please (other variables have descriptions).
I would suggest...

```Quota (in GB) to set for the VM Shared Storage.```

File: templates/workspaces/base/template_schema.json [hunk 2]
```diff
@@ -7,5 +7,11 @@
     "required": [
     ],
     "properties": {
+      "shared_storage_quota": {
+        "$id": "#/properties/shared_storage_quota",
+        "type": "integer",
+        "title": "Shared Storage Quota'",
+        "description": "Quote (in GB) to set for the VM Shared Storage."
```
Comment (martinpeck) [inline templates/workspaces/base/template_schema.json:14]: Typo `Quote` should be `Quota`

**Review by martinpeck — CHANGES_REQUESTED** [review]

File: templates/workspaces/base/terraform/variables.tf (hunk 1 above)
Comment (oliver7598) [reply templates/workspaces/base/terraform/variables.tf:11]: Changes implemented

**Review by martinpeck — APPROVED** [review]
LGTM. Thanks!

**Comment by martinpeck** [comment]
Tests have been previously run (3 times)

**Pull request merged**